internal/output/sarif.go

```diff
@@ -29,7 +29,7 @@ const SARIFTemplate = `
 **Your dependency is vulnerable to [{{.ID}}](https://osv.dev/vulnerability/{{.ID}})** 
 {{if gt (len .AliasedVulns) 1 -}}
 (Also published as: {{range .AliasedVulns -}} {{if ne .ID $.ID}} [{{.ID}}](https://osv.dev/vulnerability/{{.ID}}) {{end}}{{end}})
-{{- end}}.
+{{- end}}
```

Comment:
I think we still want to keep the period. The issue of it being on a new line is because the if statement on line 30 does not have a `-`. E.g. `{{- if gt (len ...` to remove the newline/whitespace before it.